packages/atomic/src/components/search/result-template-components/atomic-quickview-modal/atomic-quickview-modal.tsx

```diff
@@ -169,6 +221,21 @@ export class AtomicQuickviewModal implements InitializableComponent {
     return this.bindings.engine.state.search.response.termsToHighlight;
   }
 
+  private handleHighlightsScripts() {
+    if (this.highlightKeywords.highlightNone) {
+      this.enableHighlights();
+    } else {
+      this.disableHighlights();
+    }
```

Comment:
I'm kind of confused by this condition 🤔 is it inverted?

Reply:
forgot it when I inverted the meaning of the flag per your suggestion.

Fixing